cohere/response.py

```diff
@@ -66,5 +66,21 @@ def __repr__(self) -> str:
         output = f'cohere.{type(self).__name__} {{\n{contents}}}'
         return output
 
-    def feedback(self, feedback: str, accepted: bool) -> Feedback:
-        return self.client.feedback(id=self.id, feedback=feedback, accepted=accepted)
+    def feedback(self, good_generation: bool, feedback: str = "") -> Feedback:
```

Comment:
This method is generation specific, can we put in Generation class, not CohereObject?